clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -292,6 +292,9 @@ def err_drv_fsycl_with_c_type : Error<
 def warn_drv_sycl_offload_target_duplicate : Warning<
   "SYCL offloading target '%0' is similar to target '%1' already specified; "
   "will be ignored">, InGroup<SyclTarget>;
+def warn_drv_sycl_deprecated_triple_component : Warning<"SYCL offloading target"
+  " triple '%0' component is deprecated; will be ignored.">,
```

Comment:
Diagnostic is ambiguous as to what is going on.  Is the whole triple ignored?  Or is just the `sycldevice` component ignored?  Maybe add the replacement triple to the diagnostic to clarify.

Reply:
Does https://github.com/intel/llvm/pull/3929/commits/fcde634929af08292a6abb4891ca1083bcd46630 make it better?